jl4/src/L4/Syntax.hs

```diff
@@ -398,11 +398,11 @@ data Comment = MkComment Anno [Text]
   deriving stock (Show, Eq, GHC.Generic)
   deriving anyclass (SOP.Generic, ToExpr, NFData)
 
-data Nlg = MkNlg Anno [Text]
+data Nlg = MkNlg Anno Text
```

Comment:
Makes sense to me, but I'm worried how this will conflict with @fendor's work.

Reply:
I hope not, but I also don't think it will.

Reply:
It conflicts, but it doesn't matter, I am completely redoing the type any way.